GDJS/GDJS/Events/CodeGeneration/ObjectCodeGenerator.cpp

```diff
@@ -170,15 +173,15 @@ gdjs.registerObject("EXTENSION_NAME::OBJECT_NAME", CODE_NAMESPACE.RUNTIME_OBJECT
 gd::String ObjectCodeGenerator::GenerateInitializePropertyFromDataCode(
     const gd::NamedPropertyDescriptor& property) {
   return gd::String(R"jscode_template(
-    this._objectData.content.PROPERTY_NAME = objectData.content.PROPERTY_NAME !== undefined ? objectData.content.PROPERTY_NAME : DEFAULT_VALUE;)jscode_template")
+    this._objectData.PROPERTY_NAME = objectData.content.PROPERTY_NAME !== undefined ? objectData.content.PROPERTY_NAME : DEFAULT_VALUE;)jscode_template")
```

Comment:
Was using `content` an error?

Reply:
It was kind of working because it used the member of the super class but this instance was passed by constructor and could maybe be shared by all object instances.